node/pkg/alephium/watcher.go

```diff
@@ -478,17 +482,29 @@ func isEventConfirmed(
 	eventBlockHeader *sdk.BlockHeaderEntry,
 	currentTs int64,
 	currentHeight int32,
+	isMainnet bool,
 ) bool {
 	consistencyLevel := event.msg.consistencyLevel
 	if eventBlockHeader.Height+int32(consistencyLevel) > currentHeight {
 		logger.Debug("not enough block confirmations", zap.String("txId", event.TxId), zap.Uint8("consistencyLevel", consistencyLevel))
 		return false
 	}
 
-	duration := int64(consistencyLevel) * BlockTimeMs
+	duration := getConfirmationDuration(isMainnet, event.msg.IsTransferTokenVAA(), consistencyLevel)
```

Comment:
Q: Shall we check `isMainnet` here, we probably will set up a testnet bridge again soon (to test new integrations).

Reply:
If we update the `minimalConsistencyLevel` in the contract to 205, we won't need the `getConfirmationDuration` function anymore. We only need to wait for `consistencyLevel * 16s`, as the contract will ensure that the `consistencyLevel` is not less than 205.